.travis.yml

```diff
@@ -8,8 +8,3 @@ node_js:
 cache:
   directories:
     - node_modules
-
-script:
-  - npm run lint
-#  - npm run test
-  - npm run doc
```

Comment:
Why remove the scripts, you want to remove Travis?

Reply:
No it is running with travis default (npm test).

Reply:
OK, thanks :-)